picky-asn1-x509/src/algorithm_identifier.rs

```diff
@@ -744,8 +754,18 @@ pub enum AesParameters {
 
 #[derive(serde::Serialize, serde::Deserialize, Debug, PartialEq, Eq, Clone)]
 pub struct AesAuthEncParams {
-    nonce: OctetStringAsn1,
-    icv_len: IntegerAsn1,
+    pub(crate) nonce: OctetStringAsn1,
+    pub(crate) icv_len: IntegerAsn1,
```

Comment:
question: Is this change really necessary?

Reply:
I need it only for future tests to construct the struct field by field.
I don't want to make them fully public because otherwise the user can set the invalid data

Reply:
I can replace this change with a constructor method, but I don't see any big difference between `pub(crate)` and the constructor

Reply:
Oh, I just got an idea. I can introduce this change in future PR.
It's not necessary for now 😅 
[`96979d4` (#320)](https://github.com/Devolutions/picky-rs/pull/320/commits/96979d494f4f6b80284798cee52c2ea26553404a)